### DISCUSSION TITLE
Tokenize, modify, and output new text

### DISCUSSION THREAD

**Discussion by cathrinevaage** [winkjs/wink-nlp#75] [Help on coding]
Hello.
I'm trying to create a program that will take an English text, analyse and tokenise it, perform some transforms on the tokens based on the analysis, and then spit out the a new text with those transforms. 
So far I've managed to use `doc.tokens.out(its.pos)` to get the grammatical analysis I need, but that's as far as I've got. 
Is this tool suited for what it is I'm trying to achieve?

**Comment by rachnachakraborty** [comment]
Hi @cathrinevaage,

Thanks for using winkNLP.

Would be great if you can share more about the usecase you are working on for us to help you further.

Cheers,
Rachna
